obsah/__init__.py

```diff
@@ -242,6 +242,8 @@ def allow_extra_vars():
         """
         Whether to allow --extra-vars parameter to be automatically added
         """
+        if value := os.environ.get('OBSAH_ALLOW') is not None:
```

Comment:
```suggestion
        if value := os.environ.get('OBSAH_ALLOW_EXTRA_VARS') is not None:
```

don't blindly copy my suggestion ;)